industrial_robot_client/include/industrial_robot_client/joint_trajectory_action.h

```diff
@@ -176,6 +176,33 @@ class JointTrajectoryAction
    */
   static const double WATCHDOG_PERIOD_;// = 1.0;
 
+  /**
+   * \brief Should the action server abort goals if the OEM server
+   * program reports a problem?
+   *
+   * This is a backwards compatibility 'tunable knob'.
+   *
+   * Set this to true to keep the old behaviour.
+   *
+   * This is configurable, as it's possible drivers for certain robots
+   * cannot accurately report controller status (ie: have UNKNOWNs in
+   * the RobotStatus messages they publish).
+   */
+  bool ignore_motion_server_error_;
```

Comment:
```suggestion
  bool ignore_motion_server_error_ = false;
```
I think it's already implicitly initialized to that but I'd prefer having this explicitely